File: src/lib/software/razer_cortex.py
from src.lib.software import BaseSoftware

class RazerCortex(BaseSoftware):
    def __init__(self):
        super().__init__()

        self.download_name = 'RazerCortexInstaller.exe'

    def resolve_download_url(self):
        self.url_resolved.emit('https://rzr.to/cortex-download')

File: packages/app/src/services/software/definitions/razerCortex.ts
import { ok } from 'neverthrow';
import { SoftwareCategory } from 'shared';
import type { ISoftwareDefinition } from 'shared';

export default class implements ISoftwareDefinition {
	public key = 'razer-cortex';
	public name = 'Razer Cortex';
	public category = [SoftwareCategory.Gaming];
	public downloadName = 'RazerCortexInstaller.exe';
	public requiresAdmin = true;
	public icon = 'razer-cortex.png';
	public homepage = 'https://razer.com/cortex';

	public async resolveDownloadUrl() {
		return ok('https://rzr.to/cortex-download');
	}
}
